cloud/src/meta-service/meta_service_schema.cpp

```diff
@@ -331,6 +340,12 @@ void read_schema_from_dict(MetaServiceCode& code, std::string& msg, const std::s
     LOG(INFO) << "Get schema_dict, column size=" << dict.column_dict_size()
               << ", index size=" << dict.index_dict_size();
 
+    // Return dict, let backend to fill schema with dict info
+    if (schema_op == GetRowsetRequest::RETURN_DICT) {
+        response->mutable_schema_dict()->Swap(&dict);
```

Comment:
If the dict is very large, the response is memory consuming. 
Will it still be better than before, if only small ratio of dict is usefull for the response rowsets?

Reply:
for most of cases the large  ratio of dict is usefull , so the optimization is worthy of